web/src/rest.js

```diff
@@ -0,0 +1,14 @@
+
+import { RestClient } from '@girder/components/src';
+
+const apiRoot = process.env.VUE_APP_API_ROOT || 'http://localhost:8080/api/v1';
+const girderRest = new RestClient({ apiRoot, setLocalCookie: true });
+
+const loggedIn = () => !!girderRest.user;
```

Comment:
I think it may be helpful to have these as reactive properties within Vuex, but we can move them back to a separate module if / when appropriate.